CONTRIBUTING.md

```diff
@@ -110,7 +110,7 @@ Steps:
 
 ### Bump the lib Version
 
-Check the last release number at https://pypi.org/manage/project/scanapi/releases/
```

Comment:
Any special reason for removing the url? Or was it just a mistake?

Reply:
opps mistake :)